conda_smithy/configure_feedstock.py

```diff
@@ -1482,6 +1482,7 @@ def _load_forge_config(forge_dir, exclusive_config_file):
         "bot": {"automerge": False},
         "conda_forge_output_validation": False,
         "private_upload": False,
+        "secrets": ["BINSTAR_TOKEN"],
```

Comment:
Can you make this an empty list and add this later? Otherwise people would have to add this.

Reply:
That makes sense. Fixed.